expo/scheduler/views/export_calendar.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponse
+from django.views.decorators.cache import never_cache
+from scheduler.functions import calendar_export
+from gbe.functions import get_current_conference_slug
+import csv
+
+
+@login_required
+@never_cache
+def export_calendar(request):
+    '''
+    View to export calendars, wrapped around calendar_export function.
+    Used to allow importing of the calendar information into other
+    applications, such as Guidebook.
+    '''
+    #  TODO: Add ability to filter on a users schedule for things like
+    #  volunteer shifts.
+
+    conference = request.GET.get('conference', None)
+    cal_format = request.GET.get('cal_format', 'gbook')
+    event_types = request.GET.get('event_types', 'All')
+    day = request.GET.get('day', None)
+
+    if day == 'All':
+        day = None
+
+    if cal_format == 'ical':
+        filetype = 'ics'
+    else:
+        filetype = 'csv'
+
+    slug = get_current_conference_slug()
```

Comment:
Could this be the selected conference instead of just get current conference slug?  if i pull a gbe2016 calendar, it gets labeled gbe2017 which confused the heck out of me.